app/frontend/src/components/forms/ExportSubmissions.vue

```diff
@@ -1,269 +1,309 @@
-                <div class="subTitleObjectStyle mr-1">
-                  Select the export template
+                        <v-data-table
+                          :headers="headers"
+                          :search="inputFilter"
+                          show-select
+                          hide-default-footer
+                          v-model="selected"
+                          disable-sort
+                          :items="FILTER_HEADERS"
+                          item-key="name"
+                          height="300px"
+                          mobile
+                          disable-pagination
+                          fixed-header
+                          class="grey lighten-5 mt-3 submissions-table"
+                        />
+                      </v-col>
+                    </v-row>
+                  </v-col>
+                </v-row>
+              </v-col>
+            </v-row>
+            <v-row class="mt-4">
+              <v-col>
+                <p class="subTitleObjectStyle">Submission Date</p>
+                <v-radio-group v-model="dateRange" hide-details="auto">
+                  <v-radio label="All" :value="false">
+                    <template v-slot:label>
+                      <span class="radioboxLabelStyle">All</span>
+                    </template>
+                  </v-radio>
+                  <v-radio label="Select Date range" :value="true">
+                    <template v-slot:label>
+                      <span class="radioboxLabelStyle">Select date range</span>
+                    </template>
+                  </v-radio>
+                </v-radio-group>
+              </v-col>
+            </v-row>
+            <v-row class="mt-5">
+              <v-col>
+                <div v-if="dateRange">
+                  <v-row>
+                    <v-col cols="12" sm="6" offset-sm="0" offset-md="1" md="4">
+                      <v-menu
+                        v-model="startDateMenu"
+                        data-test="menu-form-startDate"
+                        :close-on-content-click="true"
+                        :nudge-right="40"
+                        transition="scale-transition"
+                        offset-y
+                        min-width="290px"
+                      >
+                        <template v-slot:activator="{ on }">
+                          <label>From</label>
+                          <v-text-field
+                            v-model="startDate"
+                            placeholder="yyyy-mm-dd"
+                            append-icon="event"
+                            v-on:click:append="startDateMenu = true"
+                            readonly
+                            v-on="on"
+                            dense
+                            outlined
+                          ></v-text-field>
+                        </template>
+                        <v-date-picker
```

Comment:
Why did we go with two separate date pickers with custom logic vs using the built in vuetify date picker with a range?

https://v2.vuetifyjs.com/en/components/date-pickers/#range

Reply:
This was inherited from the showcase team. We can create a separate ticket for this.